src/webgpu/api/operation/adapter/requestAdapterInfo.spec.ts

```diff
@@ -0,0 +1,55 @@
+export const description = `
+Tests various ways of calling GPUAdapter.requestAdapterInfo.
+
+TODO:
+- Find a way to perform tests with and without user activation
```

Comment:
Likely as a WPT-specific test (a standalone HTML file rather than a .spec.ts file) using testdriver.js:
https://web-platform-tests.org/writing-tests/testdriver.html